### PR TITLE
Work with web scrobbler v3

### PR DESCRIPTION
**Pull request by yayuyokitano**
A bit of a "quick and dirty job" here, but this moves to using TS and tsx to fetch connectors from v3, as v3 is written in TS.
Works perfectly fine on my end when I use my own fork as the source.
Linting is not setup right with this at all currently, and the any is a little yucky but it works.
Probably should not be merged until v3 is merged to master of main repo anyway.

### CODE REVIEW

**Review by inverse — APPROVED** [review]
Looks good :+1:

File: package.json [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
+    "@types/node-fetch": "^2.6.3",
     "eslint": "^7.14.0",
     "eslint-config-web-scrobbler": "^2.0.2",
+    "node-fetch": "^2.6.1",
     "prettier": "^2.2.0",
     "prettier-config-web-scrobbler": "^0.1.0",
-    "node-define": "^0.1.1",
-    "node-fetch": "^2.6.1",
-    "requirejs": "^2.3.6"
+    "requirejs": "^2.3.6",
```
Comment (inverse) [inline package.json:18]: Do we still need requirejs?

**Comment by inverse** [comment]
To be merged after https://github.com/web-scrobbler/web-scrobbler/pull/3605

File: package.json (hunk 1 above)
Comment (yayuyokitano) [reply package.json]: We do not, good catch

**Pull request merged**